### DISCUSSION TITLE
GoldHEN Cheats Manager v1.0.0

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/PS4CheatsManager#50] [Announcements]
### Added

- Support for Game Patches (@illusion0001)
  - List available patches
  - Enable/Disable patches
  - Update latest patches from the Web
- `MC4` cheat format support
  - List available cheats
  - Update `.mc4` cheat files from the Web
- New "File Overwrite" setting
  - Allows to overwrite current cheat files with updated ones

### Changed

- Updated networking code to `libcurl`+`polarssl` (thanks to @LightningMods)
- Visual improvements, icons, background (thanks to Chronoss)

### Fixed

- Fixed networking issues
- Fixed version detection for some "remaster" titles

<hr /><em>This discussion was created from the release <a href='https://github.com/GoldHEN/GoldHEN_Cheat_Manager/releases/tag/v1.0.0'>GoldHEN Cheats Manager v1.0.0</a>.</em>

**Comment by ssvillamor23** [comment]
Hi, thank you so much for the update.
I'm just wondering since the progress of number of cheats available is too slow compared to the ones who where using PC like art money and Cheat Engine by just saving a table. My point is, it is possible for this tables can be converted to Goldhen Cheat Manager acceptable formats. If it might be much doable since its a PC to PS4, just ignore my suggestions.. If I could only make my own with good tutorial.

**Comment by czirzone** [comment]
hope to soon be able to cheat for the latest FIFA 23 version CUSA31873 Update 1.0.7

**Comment by SELYANICH** [comment]
thank you very much for your application, convenient for simple users like me! cheats all work fine, but the patches from illusion0001 are activated in the application, but they don’t work in the games themselves and there are no messages that patches have been applied, I used ps4 pro fw9.0 goldhen 2.2.5b7 and 2.2.5b8 test.prx took here https:/ /mega.nz/file/VYQHQAYZ#Wq8yXzOV8gtrdTauwu1cGLU90eGjNjo5wG4gFoP9Q6w and uploaded to console via ps4xplorer 2

**Comment by bangizz0027** [comment]
can you update cheat on monster hunter world iceborne CUSA07708 v15.11 ?
